tests/10apidoc/13ui-auth.pl

```diff
@@ -0,0 +1,269 @@
+use JSON qw( decode_json );
+use URI::Escape;
+
+our @EXPORT = qw( wait_for_cas_request );
+
+# A convenience function which wraps await_http_request. It returns a successful
+# CAS response when queried for a particular path.
```

Comment:
it'd be nice to document what the `params` can be, but :man_shrugging:

Reply:
I added some.